### ISSUE TITLE
Options to display more comments per page & search

### ISSUE THREAD

**Issue by metro-view** [sameerdash2/comment-viewer#3]
Hi there, first of all thanks for doing this, it's what I've been looking for - clean, uncluttered and elegant. However I have 2 ideas for improvement - kinda the reason why I started looking for something like this in the first place.
- Option to click the number of comments per page to load e.g. 25, 50, 100, 200, ALL (if under a certain upper limit). I can't stand YT's uncustomisable infinite scroll feature.
- Option to search through ALL comments of a video even if they're not currently all being displayed on-screen.

Fantastic work, keep going!

**Comment by sameerdash2** [comment]
Thanks for the kind words!

> Option to click the number of comments per page to load e.g. 25, 50, 100, 200, ALL (if under a certain upper limit).

I played around with this a bit and it's certainly doable with 50, 100, 200, 500. Could easily do 1000, but then the browser starts to lag with so many comments on display. So I'll look into implementing this with up to 500.

> Option to search through ALL comments of a video even if they're not currently all being displayed on-screen.

I actually had this feature [in the past](https://github.com/sameerdash2/comment-viewer/blob/search/pics/search.png), using SQLite's FTS4 module. But over time (after several weeks of usage), it began to freeze and throw errors on 99% of searches. :( Maybe I did something wrong, but implementing search is just too difficult with the SQLite setup. The best approach for that would probably be just delivering the comments via file (#2) and let the user search themselves. That's an issue for the future.
